Assets/Scripts/Turretfunctions/TurretShootBase.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Turretshootbase : MonoBehaviour
+{
+
+    public Transform muzzle;
+    public GameObject bullet;
+    public GameObject temp;

```

Comment:
temp and shooting dont seem to be used anywhere, if that's the case they should be removed